adapter.js

```diff
@@ -57,6 +58,7 @@ export function adapter({ name, aws: { s3, sqs } }) {
   }
 
   return Object.freeze({
+    cleanup: () => interval && clearInterval(interval),
```

Comment:
I think you will have to add to the queue adapter port in order to add this method.